ui/src/layout/Layout.tsx

```diff
@@ -51,13 +51,21 @@ class Layout extends React.Component<WithStyles<'content'> & Stores<'currentUser
     private showSettings = false;
     @observable
     private version = Layout.defaultVersion;
+    private localStorageThemeKey = 'gotifyTheme';
 
     public componentDidMount() {
         if (this.version === Layout.defaultVersion) {
             axios.get(config.get('url') + 'version').then((resp: AxiosResponse<IVersion>) => {
                 this.version = resp.data.version;
             });
         }
+
+        const theme = this.darkThemeVisible ? 'dark' : 'light';
```

Comment:
Maybe consider refactoring `this.darkThemeVisible` so that we don't have to make the conversion between boolean and `"dark"|"light"` every time? :) f.ex. https://github.com/eternal-flame-AD/gotify-server/tree/review/persisttheme